### DISCUSSION TITLE
Solved First Challenge and got my first NFT!!!!!

### DISCUSSION THREAD

**Discussion by Raghav-Programmer** [Cyfrin/foundry-full-course-cu#331] [General]
I completed lesson-1 and solved the challenge...Can't wait to dive deeper into the Web3 world....

![#1 NFT](https://github.com/Cyfrin/foundry-full-course-f23/assets/113235744/39b4c640-b03b-4e6b-b680-04faf474a2f6)

**Comment by shikhar229169** [comment]
Congo 🎉🎉🎉
By the way, you don't have to hide the contract address or the token id of the nft, they are not private stuffs.

**Reply by Raghav-Programmer** [reply]
Thanks......oh ok I thought they were private stuff
